docs/plyql.md

```diff
@@ -180,24 +180,24 @@ Plyql has an option `--interval` (`-i`) that automatically filters time on the l
 It is useful if you do not want to type out a time filter.
 
 ```sql
-plyql -h 192.168.60.100:8082 -i P1Y -q "
+plyql -h 192.168.60.100:8082 -i P1Y -q '
 SELECT page as pg, 
 COUNT() as cnt 
 FROM wikipedia 
 GROUP BY page 
 ORDER BY cnt DESC 
 LIMIT 5;
-"
+'
 ```
 
 To get a breakdown by time the `TIME_BUCKET` function can be used:
 
 ```sql
-plyql -h 192.168.60.100:8082 -i P1Y -q "
+plyql -h 192.168.60.100:8082 -i P1Y -q '
 SELECT SUM(added) as TotalAdded 
 FROM wikipedia 
 GROUP BY TIME_BUCKET(__time, PT6H, 'Etc/UTC');
```

Comment:
you also need to convert the inner single quotes to double quotes 'Etc/UTC' => "Etc/UTC"